src/app/modules/main/stickers/models/sticker_pack_list.nim

```diff
@@ -123,6 +124,17 @@ QtObject:
     let packInfo = self.packs[self.packIdToRetrieve]
     result = newQVariant(packInfo.stickers)
 
+  # We cannot return QVariant from the proc which has arguments.
+  # First findStickersById has to be called, then getFoundStickers
```

Comment:
:dizzy_face: that's unfortunate

Reply:
Yes, I was dealing with this and thinking: what the hell !
And I found this: https://github.com/status-im/status-desktop/blob/master/src/app/modules/main/view.nim#L166

Reply:
Sad limitation